docs/NativeModulesAndroid.md

```diff
@@ -248,3 +248,15 @@ var ScrollResponderMixin = {
     this.props.onKeyboardWillShow && this.props.onKeyboardWillShow(e);
   },
 ```
+
+You can also directly use the `DeviceEventEmitter` module to listen for events.
+
+```js
+...
+componentWillMount: function() {
+  DeviceEventEmitter.addListener('keyboardWillShow', function(e: Event) {
```

Comment:
I'm not very familiar with `DeviceEventEmitter` - is the listener get automatically unregistered when the component is unmounted?